### ISSUE TITLE
Video memory

### ISSUE THREAD

**Issue by Yuliang-Liu** [gaopengcuhk/Unofficial-Pix2Seq#2]
Can it be trained under 12GB per batch?

**Comment by gaopengcuhk** [comment]
You can fit pix2seq into 12GB memory by decrease the image resolution in dataset/coco.py and also choose a small batch size.

**Comment by gaopengcuhk** [comment]
For a 48GB memory, you can fit 8 batches with image resolution 1333 which is my training setting. As transformer does not have a linear scaling law as CNN, you need to perform parameter fine-tuning for searching best learning rate and weight decay.

**Comment by Yuliang-Liu** [comment]
@gaopengcuhk Thank you!
